stream-chat-android-client/build.gradle

```diff
@@ -107,5 +107,8 @@ dependencies {
     androidTestImplementation Dependencies.espressoCore
     androidTestImplementation Dependencies.androidxTestJunit
 
-    dokkaPlugin(Dependencies.dokkasaurus)
+    dokkaHtmlPlugin("org.jetbrains.dokka:dokka-base:1.4.32")
```

Comment:
Could we extract it into our `buildSrc` project and then use something like `Dependencies.dokkaBase`? :thinking:

Reply:
No need to use dokka base anymore.